src/components/_LuxUniversityAccessibility.vue

```diff
@@ -70,7 +69,6 @@ export default {
     a {
       color: var(--color-white);
       background: var(--color-grayscale-darker);
-      &:hover,
```

Comment:
On hover in shade theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}